airflow/providers/docker/operators/docker.py

```diff
@@ -269,14 +269,17 @@ def _run_image(self) -> Optional[str]:
             # duplicated conditional logic because of expensive operation
             ret = None
             if self.do_xcom_push:
-                ret = self.cli.logs(container=self.container['Id']) if self.xcom_all else line.encode('utf-8')
+                if self.xcom_all:
+                    ret = self.cli.logs(container=self.container['Id'])
```

Comment:
why not strip here also?

Reply:
I agree it should probably be stripped, but my intention was to keep the current behavior when `do_xcom_push` and `xcom_all` are all true.